app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = current_user.likes.new(like_params)
+    redirect_to user_post_path(user_id: current_user.id, id: @post.id) if @like.save
```

Comment:
- [ ] Kudos on using `current_user` here 👏🏻

Reply:
Thank you @ShoiraTa 
I will refactor my code